test/e2e/upgradetest/upgrade_test.go

```diff
@@ -34,7 +35,11 @@ func TestResize(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	time.Sleep(10 * time.Second)
+	tprAPIPath := fmt.Sprintf("/apis/%s/%s/namespaces/%s/clusters", spec.TPRGroup, spec.TPRVersion, testF.KubeNS)
+	if err = e2eutil.WaitForAPIPath(testF.KubeCli, tprAPIPath, 10*time.Second); err != nil {
```

Comment:
Such method already exists -- `k8sutil.WaitEtcdTPRReady()`.

Please use that one. Potentially we can make the name better.